boards/apollo3/lora_things_plus/src/main.rs

```diff
@@ -472,7 +472,7 @@ unsafe fn setup() -> (
             humidity,
             air_quality,
             scheduler,
-            systick,
```

Comment:
ugh—can we set this one to the opposite? Missing trailing commas on arbitrarily-last entries just lead to noise in the diff and history; I think these should be required.

Reply:
I'm actually a little confused as well. I thought that rustfmt enforced having the trailing `,`. I wonder if this has something to do with the macro?

The clippy lint is just checking that the fields match the definition in order.

Reply:
This might be relevant: https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#trailing_comma